thoth/common/openshift.py

```diff
@@ -1643,6 +1661,10 @@ def get_mi_repositories_and_organizations(self,) -> Tuple[List[str], List[str]]:
         )
         return (repos, orgs)
 
+    def get_mi_kebechet_repositories(self):
```

Comment:
This method is not taken from common, it is a query from the database that needs to be created :)

Reply:
yes, but cannot the query be done inside common using e.g. `psycopg2` module? (I am still trying to recreate the postgresql database locally...)

Reply:
common and storages are two different libraries and their behavior is separated, moreover, storages already depend on thoth-common, you would create a cycle in dependencies that cannot be solved. We should use libraries for their purposes. common to handle kubernetes, openshift and storages to handle database. 

The component that wants to schedule something uses common, the component the wants to query something uses storages. So in the component both libraries need to be direct dependencies :)

Reply:
> yes, but cannot the query be done inside common using e.g. `psycopg2` module? (I am still trying to recreate the postgresql database locally...)

What issues are you experiencing?

Reply:
> The component that wants to schedule something uses common, the component the wants to query something uses storages. So in the component both libraries need to be direct dependencies :)

yes alright, will move the method to storages then :+1: